artacle-swagger.yaml

```diff
@@ -314,6 +314,281 @@ paths:
         '500':
           $ref: "#/components/responses/InternalServerError"
 
+#  Collection Traits
+  /collections/{cid}/traits/v1:
+    get:
+      tags:
+        - Traits
+      summary: Get traits information and valuation within a specific collection. All prices and volumes by default are provided in the native collection currency (i.e. ETH for Ethereum)
+      description: Returns all collection traits.
+      parameters:
+        - name: cid
+          in: path
+          required: true
+          description: Artacle collection ID as returned by the /collections/ endpoint
+          schema:
+            type: integer
+            example: 78
+        - name: apiKey
+          in: query
+          required: true
+          description: API access key
+          schema:
+            type: string
+        - name: traitName
+          in: query
+          required: false
+          description: Query only traits with certain name
+          schema:
+            type: string
+            example: Color
+        - name: sortBy
+          in: query
+          required: false 
+          description: Sort result by one of criteria
+          schema:
+            type: string
+            enum:
+              - valuationAllTime
+              - valuationRecent
+              - floor
+              - totalSalesVolume
+              - totalAmountOfSales
+            default: valuationAllTime
+            example: valuationAllTime
+        - name: sortingDirection
+          in: query
+          required: false
+          description: "Sorting direction: ascending, descending"
+          schema:
+            type: string
+            enum:
+              - asc
+              - desc
+            default: desc
+            example: desc
+        - name: offset
```

Comment:
Please, let's use all consistent descriptions for offset, limit and apiKey, as it used to be in other methods:
![image](https://github.com/artacle/api-docs/assets/47601437/7645e854-f920-4b45-ac5f-70143092c662)

Reply:
fixed